lib/strategy/linear.js

```diff
@@ -0,0 +1,41 @@
+//      Copyright (c) 2012 Mathieu Turcotte
+//      Licensed under the MIT license.
+
+var util = require('util');
+var precond = require('precond');
+
+var BackoffStrategy = require('./strategy');
+
+// Linear backoff strategy.
+function LinearBackoffStrategy(options) {
+    BackoffStrategy.call(this, options);
+    this.backoffDelay_ = 0;
+    this.nextBackoffDelay_ = this.getInitialDelay();
+    this.factor_ = LinearBackoffStrategy.DEFAULT_FACTOR;
+
+    if (options && options.factor !== undefined) {
```

Comment:
Factor doesn't seem to be the right name anymore since it's linear. Slope is probably closer to what you're looking for.

https://en.wikipedia.org/wiki/Linear_equation